site/content/3.11/release-notes/version-3.11/whats-new-in-3-11.md

```diff
@@ -1011,6 +1011,30 @@ attempt to create an additional database fails with error
 if other databases are dropped first. The default value for this option is
 unlimited, so an arbitrary amount of databases can be created.
 
+### Cluster-internal connectivity checks
+
+<small>Introduced in: v3.11.5</small>
+
+This feature makes Coordinators and DB-Servers in a cluster periodically send
+check requests to each other, in order to check if all servers can connect to
```

Comment:
Grammarly marks "to check if" as a problem 🤔 
Servers as a term is a bit ambiguous, I think nodes is better in this context.
```suggestion
check requests to each other, in order to see if all nodes can connect to
```